dix/req.c

```diff
@@ -1595,7 +1595,7 @@ static void dix_rop_completed(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 {
 	struct m0_dix_req     *req = ast->sa_datum;
 	struct m0_dix_rop_ctx *rop = req->dr_rop;
-	struct m0_dix_rop_ctx *rop_del_phase2;
+	struct m0_dix_rop_ctx *rop_del_phase2=NULL;
```

Comment:
```
struct m0_dix_rop_ctx *rop_del_phase2 = NULL;
```

add space between operand and operator.

Reply:
Fixed with proper formatting with spaces